source/version.js

```diff
@@ -57,3 +57,10 @@ module.exports.validate = version => {
 		throw new Error('Version should be a valid semver version.');
 	}
 };
+
+module.exports.versionSatisfiesRequirement = (dependency, v) => {
```

Comment:
```suggestion
module.exports.versionSatisfiesRequirement = (dependency, version) => {
```

To make the function more readable (I originally used `v` instead of `version` to avoid any conflicts in other files where we do `const version = require('../version')`, but in the `version.js` file itself this isn't an issue).